webapp/channels/src/components/post_emoji/post_emoji.tsx

```diff
@@ -22,14 +24,22 @@ const PostEmoji = ({name, imageUrl}: Props) => {
     }
 
     return (
-        <span
-            alt={emojiText}
-            className='emoticon'
+        <WithTooltip
+            id='postEmoji__tooltip'
```

Comment:
@larkox I'm not sure if you're aware of this, but this ID prop doesn't actually need to be unique with how we're using React Bootstrap for it. I thought that this needed to be unique since it's passed through to the DOM, but since only one `WithTooltip` ever renders its tooltip at a time, it doesn't end up needing to be unique